assets/js/wpjm-stats.js

```diff
@@ -0,0 +1,78 @@
+/* global job_manager_stats */
+( function () {
+	// From https://youmightnotneedjquery.com/#ready
+	function ready( fn ) {
+		if ( document.readyState !== 'loading' ) {
+			fn();
+		} else {
+			document.addEventListener( 'DOMContentLoaded', fn );
+		}
+	}
+
+	// Cookie funcs are copied verbatim from http://www.quirksmode.org/js/cookies.html, tweaked to include path.
+	function createCookie( name, value, days, path ) {
```

Comment:
Since we are only using the data from JS, I think we'd better off with [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) instead of cookies.
- Cookies get sent to the server on every request
- Way more complicated API to set and read them